### PR TITLE
[ZEPPELIN-1122] BugFix for repositories snapshot is always set to true.

### PR DESCRIPTION
**Pull request by astroshim**
### What is this PR for?
This PR fixes a bug about snapshot setting and select box(true,false) html.

### What type of PR is it?
Bug Fix | Improvement

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1122

### How should this be tested?
1. go Interpreter menu.
2. click ```repository information``` button that is right top.
3. click plus button (to Add New Repository)
and refer to screenshot.

### Screenshots (if appropriate)
- before
b

- after
a

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/AbstractDependencyResolver.java [hunk 1]
```diff
@@ -64,8 +64,8 @@ public void addRepo(String id, String url, boolean snapshot) {
     synchronized (repos) {
       delRepo(id);
       RemoteRepository rr = new RemoteRepository(id, "default", url);
-      rr.setPolicy(snapshot, new RepositoryPolicy(
-          true,
+      rr.setPolicy(true, new RepositoryPolicy(
```
Comment (minahlee) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/AbstractDependencyResolver.java]: According to [sonatype aether api doc](http://sonatype.github.io/sonatype-aether/apidocs/org/sonatype/aether/repository/RemoteRepository.html#setPolicy(boolean, org.sonatype.aether.repository.RepositoryPolicy)), original code seems to be the correct. Did I miss something?

Comment (astroshim) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/AbstractDependencyResolver.java]: @minahlee Oh. That part is not the problem. I'll fix that.

**Comment by astroshim** [comment]
The name "Snapshot" that presented on popup before is not snapshot information but the "Enable" flag for the associated repository information.
So I removed the snapshot from popup.

**Comment by bzz** [comment]
Thank you for fixing!  Looks like CI fails dues to https://github.com/npm/npm/issues/13284

Looks good to me, merging if there is no more discussion

**Comment by bzz** [comment]
Looks great to me, thank you, CI passes now.

**Comment by bzz** [comment]
@astroshim can you please next time set "Fix Version" in JIRA and make sure issue is assigned to you? 
This helps release manager a lot!

**Comment by astroshim** [comment]
@bzz oh I see. Thank you reminding me.

**Pull request closed without merging**